src/runners/ingest_runner.py

```diff
@@ -15,6 +24,21 @@ def main():
         except Exception as e:
             log.error(f"failed to run {name}", e)
 
+    if connection_table:
+        tables = db.fetch(f"SHOW TABLES LIKE '{connection_table}' IN DATA")
+    else:
+        tables = db.fetch(CONNECTION_TABLE_QUERY)
+
+    for table in tables:
+        log.info(f"Starting {table['name']}")
+        options = yaml.load(table['comment'])
+        if 'source' in options:
+            for option in options:
+                if option.endswith('secret'):
+                    options[option] = kms.decrypt_if_encrypted(options[option])
+            connection_module = importlib.import_module(f"connectors.{options['source']}")
+            connection_module.populate(options['name'], options)
```

Comment:
`populate` makes it sound like it's wiping out the existing data. let's keep the `ingest` terminology from TDD